app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,57 @@
+<div class="align_center slogan">
+  <h1 class="custom_size">WizList</h1>
```

Comment:
Name for a class `custom_size` without any meaning. Again, this rule should be included inside the `.slogan` like so:

```sass
.slogan {
  ...
  h1 {
    font-size: 1.6em;
  }
}
```